string status
float32 target_value
float32 current_value
